.github/workflows/ci-actions.yml

```diff
@@ -122,7 +122,7 @@ jobs:
 
     services:
       keycloak:
-        image: quay.io/keycloak/keycloak:10.0.1
+        image: quay.io/keycloak/keycloak:10.0.0
```

Comment:
how does this relate?

There's another change to keycloak version below.

Reply:
good catch, must have been a bad auto-merge when I rebased -- will fix this